database.js

```diff
@@ -12,6 +10,14 @@ const getAllBooks = () => {
   return database.any(sql)
 }
 
+const createBook = (title, author, image_url, description) => {
+  const sql = `INSERT INTO books (title, author, image_url, description) VALUES ($1, $2, $3, $4) RETURNING *`
```

Comment:
We don't need to use backticks here since this is not an interpolated string - I prefer to use single or double quotes to signal that this is a plain ol' string.